sktime/classification/compose/_ensemble.py

```diff
@@ -386,6 +387,21 @@ class in a leaf.
 
         return np.sum(all_proba, axis=0) / len(self.estimators_)
 
+    def _validate_X_predict(self, X):
+        n_features = X.shape[1] if X.ndim == 2 else 1
+        if self.n_columns != n_features:
+            raise ValueError("Number of features of the model must "
+                             "match the input. Model n_features is %s and "
+                             "input n_features is %s "
+                             % (self.n_columns, n_features))
+        return X
+
+    def apply(self, X):
+        raise NotImplementedError()
+
+    def decision_path(self, X):
+        raise NotImplementedError()
```

Comment:
Could you move these functions to the `BaseTimeSeriesForest` and remove the ones that are there at the moment that's something I overlooked when merging Ayushmaan's changes, the ones that are there at the moment probably don't work, because they come from sklearn, so it's better to raise an `NotImplementedError()` -- thanks!